src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+import uniqueId from 'lodash.uniqueid'
+import './FormGroup.css'
+import { Disabled } from 'interfaces'
+
+interface FormGroupProps extends Disabled {
+  className?: string
+  hideFormGroup?: boolean
+}
+
+export const FormGroup: React.FC<FormGroupProps> = ({
+  className = undefined,
+  disabled = false,
+  hideFormGroup = false,
+  children = '',
```

Comment:
setting `className = undefined` is specifically for when using the package `classNames`. In this case, className is unused here.
`children` is a prop thats managed by react, over in [Button.tsx](https://github.com/metruzanca/NightTab/blob/0e110408c3e9d64df4b15cf1369d3355cb9a54c1/src/components/Button/Button.tsx#L31) i set it as a string because I wanted the default to be a specific line of text, but if a component isn't passed any children then children will be null and won't render to the dom.

```suggestion
  disabled = false,
  hideFormGroup = false,
  children,
```